tools/testing-server/routes/mock-apis.js

```diff
@@ -200,13 +201,19 @@ module.exports = fp(async function (fastify, testServer) {
   fastify.post('/formdata', {
     compress: false
   }, async (request, reply) => {
+    const results = new FormData()
+
     try {
       assert.strictEqual(request.body.name.value, 'bob')
       assert.strictEqual(request.body.x.value, '5')
-      reply.send('good')
+
+      results.append('result', 'good')
     } catch (e) {
-      reply.send('bad')
+      results.append('result', 'good')
```

Comment:
might need a cliff-notes as to what this is doing

Reply:
oops that should be `bad`